### DISCUSSION TITLE
List selection

### DISCUSSION THREAD

**Discussion by JoshTango** [OrchardCMS/OrchardCore#15876] [Q&A]
I can not see how you add content items to a list in the editor.  In Orchard CMS there was a popup dialog and I could select one or many items.  I don't see that in Orchard Core.  Is there a document showing how to do this?  Preferably one with screenshots

**Comment by sebastienros** [comment]
You can't add existing ones, just add new ones right now.

**Comment by JoshTango** [comment]
But in the future we can add existing ones?  If so how many sprints away?  Also I would like the ability to have content items belong to more than one list.  For example I want to create a content type called LearningModule which has a list of Lessons.  But a Lesson could belong to more than one LearningModule.

**Comment by sebastienros** [comment]
That's what list is doing. But in this case you might not need the List content type, just a field that will reference other content items, and this way we can keep the List as it is. This will be available soon I assume as we have someone already working on it.

**Comment by JoshTango** [comment]
Yes I noticed there is a content picker  field in the old version but  the content items were displayed horizontally rather than displayed in a nice list.  Maybe the new core version will require making a template for how the children are  displayed

**Comment by JoshTango** [comment]
Content picker field will not work for my needs.  I need to pick existing items and be able to order them how I want.  In the list so far I can not see how to pick items.  I am watching and learning your new cms.

**Comment by deanmarcussen** [comment]
see also https://github.com/OrchardCMS/OrchardCore/issues/3667
